### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by johannasjo**
**To view the styles added in the CSS-file:**
npm run dev

or for continuos compilation:

npm run watch 

**To add CSS-file in blade-file add in head:**
<link rel="stylesheet" href="{{ asset('/css/app.css') }}">

the path _might_ be different depending on which blade, but should refer to the CSS-folder underneath **public**

### CODE REVIEW

**Comment by axelra82** [comment]
@johannasjo 
[Dev deployment](https://imdb-clone-footer-2d7nlp0byhc3.herokuapp.com) only shows "Home" with no styling 🤔 

Not sure if I missed something, did a pull and ran footer branch, same issue... also, not seeing the default home page in main either. Was this merged in main?

It's probably just med on a regular Sunday 🤪  We can look into all of this during our weekly tomorrow 👍

**Comment by johannasjo** [comment]
the views should work now, please test that it works for you too and looks alright :) I am working on the home-view, it's not done at all but the start is in the home branch :)

**Comment by axelra82** [comment]
Everything looks good and is working now... maybe we should proceed with a merge, just so we can keep our main up to date 😃

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -23,6 +23,9 @@ public function register()
      */
     public function boot()
     {
-        //
+        if (config('app.env') === 'production') {
+            \URL::forceScheme('https');
```
Comment (axelra82) [inline app/Providers/AppServiceProvider.php:27]: @johannasjo did we need this? Wasn't the secure_assets function enough? 🤔

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "scripts": {
+        "heroku-postbuild": "npm run production",
```
Comment (axelra82) [inline package.json:4]: This was the solution for our Heroku deploy build 👍

File: webpack.mix.js [hunk 3]
```diff
@@ -11,7 +11,6 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-    .postCss('resources/css/app.css', 'public/css', [
-        //
-    ]);
+mix.js('resources/js/app.js', 'public/js').sass('resources/css/app.scss', 'public/css', [
```
Comment (axelra82) [inline webpack.mix.js:14]: Switching over to sass from postCss

**Review by axelra82 — APPROVED** [review]
app.json is still a test. Hopefully this will solve buildpacks that are needed for review apps.

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (johannasjo) [reply app/Providers/AppServiceProvider.php:27]: I believe the secure_assets was enough :)

**Pull request closed without merging**